packages/core/types/src/auth/providers/google.ts

```diff
@@ -1,6 +1,6 @@
 export interface GoogleAuthProviderOptions {
-  clientID: string
+  clientId: string
```

Comment:
This is a breaking change but I thought it might be better to do it now than later. I don't think many people have used google auth anyway.